### PR TITLE
Add tutorial on sending logs from vector to quickwit in otel format.

### PR DESCRIPTION
**Pull request by fmassot**
Ideally, you need to read the tutorial with docusaurus as `Tabs` are not handled by GitHub markdown.

### CODE REVIEW

File: docs/guides/send-logs-from-vector-to-quickwit.mdx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+[Vector](https://vector.dev/) is an amazing piece of software (in Rust obviously) and brings a new fresh wind in the observability space.
+In this guide, we are very happy to show you how to connect it to Quickwit.
+
+## Create an index for logs
+
+Let's embrace the OpenTelemetry standard and create a index compatible with its [log data model](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md).
```
Comment (evanxg852000) [inline docs/guides/send-logs-from-vector-to-quickwit.mdx:14]:
```suggestion
Let's embrace the OpenTelemetry standard and create an index compatible with its [log data model](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md).
```

File: docs/guides/send-logs-from-vector-to-quickwit.mdx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+<Tabs>
+
+<TabItem value="docker" label="Docker">
+
+```bash
+curl -o otel-logs.yaml https://raw.githubusercontent.com/quickwit-oss/quickwit/main/config/tutorials/otel-logs/index-config.yaml
+# Create quickwit datadir.
+mkdir -p qwdata
+# Create index.
+docker run -v $(pwd)/qwdata:/quickwit/qwdata -v $(pwd)/otel-logs.yaml:/quickwit/index-config.yaml index create --index-config /quickwit/index-config.yaml
```
Comment (evanxg852000) [inline docs/guides/send-logs-from-vector-to-quickwit.mdx:65]: the image id ?

File: docs/guides/send-logs-from-vector-to-quickwit.mdx [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+Our sink here will be Quickwit ingest API `http://127.0.0.1:7280/api/v1/otel-logs/ingest`.
+To keep it simple in this tutorial, we will use a log source called `demo_logs` that generates logs in a given format. Let's choose the common `syslog` format
+(Vector does not generate logs in the OTEL format directly!) and use the transform feature to map the `syslog` format into the OpenTelemetry format.  
+
+
+```toml title=vector.toml
+[sources.generate_syslog]
+type = "demo_logs"
+format = "syslog"
+count = 100
```
Comment (evanxg852000) [inline docs/guides/send-logs-from-vector-to-quickwit.mdx:108]: I think we should increase

File: docs/guides/send-logs-from-vector-to-quickwit.mdx (hunk 2 above)
Comment (evanxg852000) [inline docs/guides/send-logs-from-vector-to-quickwit.mdx:65]:
```suggestion
docker run -v $(pwd)/qwdata:/quickwit/qwdata -v $(pwd)/otel-logs.yaml:/quickwit/index-config.yaml quickwit/quickwit index create --index-config /quickwit/index-config.yaml
```

File: docs/guides/send-logs-from-vector-to-quickwit.mdx (hunk 3 above)
Comment (fmassot) [reply docs/guides/send-logs-from-vector-to-quickwit.mdx:108]: yes absolutely, and the speed too

File: config/tutorials/otel-logs/vector.toml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+[sinks.emit_syslog]
+inputs = ["remap_syslog"]
+type = "console"
+encoding.codec = "json"
+
+[sinks.quickwit_logs]
+type = "http"
+inputs = ["remap_syslog"]
+encoding.codec = "ndjson"
+uri = "http://host.docker.internal:7280/api/v1/otel-logs/bulk"
```
Comment (evanxg852000) [inline config/tutorials/otel-logs/vector.toml:44]: I don't think this `host.docker.internal` feature works seamlessly between platforms and setup

File: docs/guides/send-logs-from-vector-to-quickwit.mdx [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+[sinks.emit_syslog]
+inputs = ["remap_syslog"]
+type = "console"
+encoding.codec = "json"
+
+[sinks.quickwit_logs]
+type = "http"
+inputs = ["remap_syslog"]
+encoding.codec = "ndjson"
+uri = "http://0.0.0.0:7280/api/v1/otel-logs/bulk"
```
Comment (evanxg852000) [inline docs/guides/send-logs-from-vector-to-quickwit.mdx:148]: I struggled a bit on this ip thingy, look up my comment

**Review by evanxg852000 — COMMENTED** [review]
Great tutorial, I can't wait to see this chained with Grafana

File: docs/guides/send-logs-from-vector-to-quickwit.mdx (hunk 5 above)
Comment (fmassot) [reply docs/guides/send-logs-from-vector-to-quickwit.mdx:148]: the IP is wrong here, should be on 127.0.0.1.

File: config/tutorials/otel-logs/vector.toml (hunk 4 above)
Comment (fmassot) [reply config/tutorials/otel-logs/vector.toml:44]: I hate having additional commands. This additional argument should do the job "--add-host=host.docker.internal:host-gateway"

**Comment by fmassot** [comment]
@evanxg852000 PTAL :)

**Review by evanxg852000 — APPROVED** [review]

**Pull request merged**